src/Field.php

```diff
@@ -508,6 +508,33 @@ public function useAlias(): bool
 
     // }}}
 
+    // {{{ Scope condition
+
+    /**
+     * Returns arguments to be used for query on this field based on the condition.
+     *
+     * @todo: value is array
+     */
+    public function getQueryArguments(Model\Scope\Condition $condition, $operator, $value): array
```

Comment:
import scope

Reply:
- Field should move under Model NS (up for discussion) so this will sort out naturally
- No, because the operator passed here has been modified already by the Condition object. 
To avoid confusion we can skip passing `$condition` altogether but it is good to have it I believe.
- will think about it
- 👍

Reply:
>     * Field should move under Model NS (up for discussion) so this will sort out naturally

then we will remove use, very simple ;-)

>     * No, because the operator passed here has been modified already by the Condition object.
>       To avoid confusion we can skip passing `$condition` altogether but it is good to have it I believe.

Hmm, I think we should create new Condition object on calling side and pass it to here

Reply:
`$condition` as an argument is not necessary and makes confusion so removed